spec.html

```diff
@@ -10,7 +10,7 @@
   }
 </script>
 <pre class=metadata>
-  title: ECMAScript® 2016 Language Specification
```

Comment:
I am fine with this (though don't have time to accept the PR now). Totally agree that this is the right thing to do for now :)

I tend agree with @ljharb that not using entities unless required (ie, encoding the input document as utf-8 and only using entities for special characters) is better. It's annoying to type but easy to copy/paste the characters from elsewhere in the document. It drastically improves the readability of the input text. But let's do it as a wholesale conversion rather than having two conventions in the doc :)